src/bindings/python/flux/job/list.py

```diff
@@ -20,9 +20,13 @@
 
 class JobListRPC(RPC):
     def get_jobs(self):
+        """Returns all jobs in the RPC"""
         return self.get()["jobs"]
 
     def get_jobinfos(self):
+        """Yields a JobInfo object for each job in its current state
+
+        :rtype: JobInfo"""
```

Comment:
prefer closing `"""` on a separate line. Also we've slowly transitioned to [Google docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style if you'd prefer that (a bit easier to read in the code)

Reply:
It looks like you left this one unaddressed @wihobbs ?